app/frontend/src/features/profile/view/UserOverview.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story } from "@storybook/react";
+import UserOverview from "features/profile/view/UserOverview";
+import users from "test/fixtures/users.json";
+
+export default {
+  component: UserOverview,
+  title: "Profile/UserOverview",
+} as Meta;
+
+const Template: Story<any> = (args) => <UserOverview {...args} />;
```

Comment:
```suggestion
const Template: Story<UserSummaryProps> = (args) => <UserOverview {...args} />;
```
Worth typing this properly if it's gonna be used as an input argument later on (line 14). You'll have to export `UserSummaryProps` too